packages/rs-drive-abci/src/query/v0/mod.rs

```diff
@@ -71,20 +71,20 @@ fn convert_key_request_type(
             let purpose_map = search_key.purpose_map.into_iter().map(|(purpose, security_level_map)| {
                 let security_level_map = security_level_map.security_level_map.into_iter().map(|(security_level, key_kind_request_type)| {
                     if security_level > u8::MAX as u32 {
-                        return Err(QueryError::Query(QuerySyntaxError::InvalidKeyParameter("security level out of bounds".to_string())));
+                        return Err(QueryError::DocumentQuery(QuerySyntaxError::InvalidKeyParameter("security level out of bounds".to_string())));
```

Comment:
I'm not sure we are querying documents here

Reply:
It seems we use query errors for identities and documents. Let's rename DocumentQuery back to Query and refactor it later

Reply:
Fixed